bitbucket/bitbucket-server-dark.user.css

```diff
@@ -825,6 +825,11 @@
     border-color: var(--border-colour);
   }
 
+  /* Approve Button Style */
+  .aui-button.approve > span {
+  	background-color: #36B37E;
```

Comment:
Why have you gone for hard-coded colours here? There are suitable background colours for buttons in the custom properties.

Reply:
Once approval has been given on a PR in Bitbucket the approval button icon background is also meant to be set to that colour. I couldn't see anywhere else that used the colour so I didn't add it as a variable.